docs/config/jest-setup.js

```diff
@@ -1,6 +1,5 @@
 import { configure } from "@testing-library/react";
 import "@testing-library/jest-dom";
-import "jest-styled-components";
```

Comment:
We don't seem to need this at all, I'm removing it because of an error while generating a snapshot of table of contents, I assume because of some invalid CSS that was most probably not caused by these changes.